editor/plugins/packed_scene_translation_parser_plugin.cpp

```diff
@@ -73,26 +82,40 @@ Error PackedSceneEditorTranslationParserPlugin::parse_file(const String &p_path,
 					Vector<String> temp;
 					Vector<Vector<String>> ids_context_plural;
 					EditorTranslationParser::get_singleton()->get_parser(extension)->parse_file(s->get_path(), &temp, &ids_context_plural);
-					parsed_strings.append_array(temp);
+					new_strings.append_array(temp);
 					r_ids_ctx_plural->append_array(ids_context_plural);
 				}
-			} else if (property_name == "filters") {
+			} else if ((node_type == "MenuButton" || node_type == "OptionButton") && property_name == "items") {
+				Vector<String> str_values = property_value;
+				// If the way items are ordered in this Nodes is ever changed, this will also need to be updated.
+				int incr_value = node_type == "MenuButton" ? 10 : 5;
```

Comment:
Maybe instead of using magic number, we could add a constant to Menu/OptionButton? (if there isn't one already...)